integrations/upgrade/index.test.ts

```diff
@@ -1600,16 +1600,16 @@ test(
     expect(await fs.dumpFiles('./src/**/*.css')).toMatchInlineSnapshot(`
       "
       --- ./src/index.css ---
-      @import 'tailwindcss/utilities' layer(utilities);
-
       /* No layer expected */
       @import './my-components.css';
 
       /* No layer expected */
       @import './my-utilities.css';
 
       /* Expecting a layer */
-      @import './my-other.css' layer(utilities);
+      @import './my-other.css' layer(components);
+
+      @import 'tailwindcss/utilities' layer(utilities);
```

Comment:
It's moved to the bottom because we replace `@import 'tailwindcss/utilities';` with the new at-rule. Currently we just drop `@import 'tailwindcss/components';` but we could replace that rule as well instead such that this import lives at the top. 🤔